notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -453,7 +447,7 @@ public boolean deleteNote(String noteType, String noteOwner, String noteName, Id
         log.error("Can't delete note '" + noteName + "'. This note does not exist.");
         return false;
       }
-    } catch (WikiException e) {
+    } catch (Exception e) {
```

Comment:
```suggestion
    } catch (WikiException e) {
```
I think here that the only catched exception is WikiException